definitions/procedures/make_sync_plans_enable.rb

```diff
@@ -0,0 +1,12 @@
+class Procedures::MakeSyncPlansEnable < ForemanMaintain::Procedure
+  for_feature :sync_plans
+  description 'enable sync plans which are disabled before upgrade'
+
+  def run
```

Comment:
After `https://github.com/iNecas/foreman_maintain/pull/29` we could have just one precodure (`UpgradeSyncPlansDisable`), which could have a parameter (such as `rollback`) to disable (and enable) the sync plans at the end